keps/sig-storage/4049-storage-capacity-scoring-of-nodes-for-dynamic-provisioning/kep.yaml

```diff
@@ -0,0 +1,45 @@
+title: Storage Capacity Scoring of Nodes for Dynamic Provisioning
+kep-number: 4049
+authors:
+  - "@cupnes"
+owning-sig: sig-storage
+participating-sigs:
+status: TBD
```

Comment:
This needs to be filled out.  It looks like you want to get the KEP merged without going through PRR review?  If so, you can set the status to "Provisional".

Reply:
That was not my intention. I am planning to undergo a PRR review.

Reply:
yes make this `implementable`